content/docs/demos/cert-manager_integration.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Cert-manager Certificate Provider"
+description: "Using cert-manager as a certificate provider"
+type: docs
+weight: 30
+---
+
+This guide demonstrates the usage of [cert-manager][1] as a certificate provider to manage and issue certificates in OSM.
+
+## Prerequisites
+
+- Kubernetes cluster running Kubernetes v1.19.0 or greater.
+- Have OSM installed.
+- Have `kubectl` available to interact with the API server.
+- Have `osm` CLI available for managing the service mesh.
+
+
+## Demo
+
+The following demo uses [cert-manager][1] as the certificate provider to issue certificates to the `curl` and `httpbin` applications communicating over `Mutual TLS (mTLS)` in an OSM managed service mesh.
+
+1. Install `cert-manager`. This demo uses `cert-manager v1.6.1`.
+    ```bash
+    kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.6.1/cert-manager.yaml
+    ```
+
+    Confirm the pods are ready and running in the `cert-manager` namespace.
+    ```console
+    kubectl get pod -n cert-manager
+    NAME                                      READY   STATUS    RESTARTS   AGE
+    cert-manager-55658cdf68-pdnzg             1/1     Running   0          2m33s
+    cert-manager-cainjector-967788869-prtjq   1/1     Running   0          2m33s
+    cert-manager-webhook-6668fbb57d-vzm4j     1/1     Running   0          2m33s
+    ```
+
+1. Configure `cert-manager` `Issuer` and `Certificate` resources required by `cert-manager` to be able to issue certificates in OSM. These resources must be created in the namespace where OSM will be installed later.
+    > Note: `cert-manager` must first be installed, with an issuer ready, before OSM can be installed using `cert-manager` as the certificate provider.
+
+    ```bash
+    # Create the namespace where OSM will be installed
+    export osm_namespace=osm-system # Replace osm-system with the namespace where OSM is installed
+    kubectl create namespace "$osm_namespace"
+
+    # Create Issuer and Certificate resources
+    kubectl apply -f - <<EOF
+    apiVersion: cert-manager.io/v1
+    kind: Issuer
+    metadata:
+      name: selfsigned
```

Comment:
I think it would be helpful to explain that we are using the selfsigned issuer to bootstrap a custom root certificate.